src/flag_gems/runtime/__init__.py

```diff
@@ -0,0 +1,21 @@
+from . import backend, commom_utils, register
+from .backend.device import DeviceDetector
+from .configloader import ConfigLoader
+
+configer = ConfigLoader()
+device = DeviceDetector()
+
+
+def get_triton_config(op_name):
+    return configer.get_triton_config(op_name)
+
+
+def to_register(*args, **kargs):
+    register.to_register(*args, **kargs)
+
+
+def get_device_fn(api_name):
+    return backend.gen_torch_device_fn(api_name)
```

Comment:
and we don't need to package the function again. import is okay.

Reply:
For future retention, it must exist